packages/suite/src/views/wallet/trading/common/TradingForm/TradingNetworkReserveBanner.tsx

```diff
@@ -0,0 +1,62 @@
+import { getNetworkReserve } from '@suite-common/trading';
+import { NetworkSymbol, getNetwork } from '@suite-common/wallet-config';
+import { selectIsNetworkReserveEnabled } from '@suite-common/wallet-core';
+import { Banner } from '@trezor/components';
+
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite/Translation';
+import { SettingsAnchor } from 'src/constants/suite/anchors';
+import { useDispatch, useSelector } from 'src/hooks/suite';
+
+interface TradingNetworkReserveBannerProps {
+    symbol: NetworkSymbol;
+    contractAddress?: string;
+    isDisabled?: boolean;
+}
+
+export const TradingNetworkReserveBanner = ({
+    symbol,
+    contractAddress,
+    isDisabled = true,
+}: TradingNetworkReserveBannerProps) => {
+    const dispatch = useDispatch();
+    const isNetworkReserveEnabled = useSelector(selectIsNetworkReserveEnabled);
+
+    const onManageClick = () => {
+        dispatch(
+            goto('settings-index', {
+                preserveParams: true,
+                anchor: SettingsAnchor.NetworkReserve,
+            }),
+        );
+    };
+
+    if (isDisabled) return null;
+
+    if (!isNetworkReserveEnabled) return null;
+    if (contractAddress) return null;
+
+    const networkReserve = getNetworkReserve(symbol, contractAddress, isNetworkReserveEnabled);
+    if (!networkReserve) return null;
+
+    const network = getNetwork(symbol);
```

Comment:
Would it be possible to show this only if `showReserveBanner` is `true` without other coniditions?

There can be bugs in compose methods, it will calcualte with reserve but this banner will have different logic and it will simply be/not be visible. It would be nice to have only "one" source of truth